endpoints/openrtb2/auction.go

```diff
@@ -1711,9 +1706,24 @@ func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson
 	return resolvedRequest, impExtInfoMap, nil
 }
 
+func setImpExtInfoMap(imp json.RawMessage, impData ImpExtPrebidData, impExtInfoMap map[string]exchange.ImpExtInfo) error {
+	impId, err := jsonparser.GetString(imp, "id")
+	if err != nil {
+		if err.Error() == "Key path not found" {
+			return errors.New("request.imp[0] missing required field: \"id\"\n")
+		}
+		return err
+	}
+	echoVideoAttributes := false
+	if impData.ImpExtPrebid.Options != nil {
+		echoVideoAttributes = impData.ImpExtPrebid.Options.EchoVideoAttrs
+	}
+	impExtInfoMap[impId] = exchange.ImpExtInfo{EchoVideoAttrs: echoVideoAttributes, Passthrough: impData.ImpExtPrebid.Passthrough}
```

Comment:
Before, we were also setting `StoredImp` value in the `ImpExtInfo` struct. Is that not needed anymore?